realm/realm-library/src/objectServer/java/io/realm/SyncManager.java

```diff
@@ -226,6 +227,15 @@ public static synchronized SyncSession getSession(SyncConfiguration syncConfigur
             sessions.put(syncConfiguration.getPath(), session);
             if (sessions.size() == 1) {
                 RealmLog.debug("first session created add network listener");
+                if(resolvedRealmURL.length > 0) {
```

Comment:
Can it be more than 1, why use varargs?

Reply:
see https://github.com/realm/realm-java/pull/5820#discussion_r173489662

Reply:
See above. I assume this can be removed now?